Documentation/TopLevelObjects/Page/Index.rst

```diff
@@ -591,9 +591,14 @@ Properties
 
         For each key the following sub-properties are available:
 
+        `attribute`
+            Sets the attribute for the meta tag. If it is not defined, the
+            default :html:`name` is used.
```

Comment:
Maybe we could be even more verbose here along the lines of:

The :html:<meta> Tag usually uses attributes like :html:<meta name="..."> (HTML5), but :html:<meta property="..."> is also valid for RDFa metadata like specifyinf OpenGraph properties. By specifying the 'attribute' key, the used attribute can be set to 'name' or 'property', defaulting to 'name'

But current version is already an improvement and I'd be happy with it, too!

Reply:
Please make a follow-up. I won't work on this PR anymore.

Reply:
Hm.